ocaml/xenops/xenops.ml

```diff
@@ -205,7 +205,7 @@ let list_devices ~xs ~domid =
 	    Xenbus.to_string_desc (Xenbus.of_string (xs.Xs.read (sprintf "%s/state" path)))
 	  with _ -> "gone" in
 	List.iter (fun device ->
-		     let fe = frontend_path_of_device ~xs device
+		     let fe = frontend_ro_path_of_device ~xs device
```

Comment:
Sorry, this should be the rw path.
This is because we use it to look up the current state of the device using the `string_of_state` function as part of a constructing a human-readable string:
see lines 203 and 214.